components/layout/display_list_builder.rs

```diff
@@ -1440,7 +1440,7 @@ impl FragmentDisplayListBuilding for Fragment {
     fn adjust_clipping_region_for_children(&self,
                                            current_clip: &mut ClippingRegion,
                                            stacking_relative_border_box: &Rect<Au>,
-                                           is_absolutely_positioned: bool) {
+                                           _is_absolutely_positioned: bool) {
```

Comment:
Oops, I submitted my own fix for this in #10469 before I saw this PR.  Sorry!  If that lands, then this commit can be dropped.